src/codegen/mod.rs

```diff
@@ -3357,6 +3404,52 @@ impl std::str::FromStr for AliasVariation {
     }
 }
 
+/// Enum for how non-Rust unions should be translated.
```

Comment:
Calling this non-rust unions, yet generating rust unions feels a bit weird... Maybe this could be `NonCopyUnionStyle` or so?